source/wp-content/themes/wporg-developer-2023/src/style/style.scss

```diff
@@ -32,17 +32,17 @@ pre {
 		padding: 20px;
 		background-color: #f7f7f7;
 		border: 1px solid var(--wp--preset--color--light-grey-1);
-		border-radius: 2px;
+		border-radius: var(--wp--custom--button--border--radius);
 		overflow: scroll;
 	}
 
 	code {
 		display: inline-block;
 		line-height: var(--wp--custom--body--extra-small--typography--line-height);
 		background: var(--wp--preset--color--light-grey-2);
-		border-radius: 2px;
-		padding-inline-start: 3px;
-		padding-inline-end: 3px;
+		border-radius: var(--wp--custom--button--border--radius);
+		padding-inline-start: 0.25em;
+		padding-inline-end: 0.25em;
```

Comment:
[This rule](https://github.com/WordPress/wporg-developer/blob/trunk/source/wp-content/themes/wporg-developer-2023/src/style/style.scss#L185) is overriding these values for inline code, eg. in the [More Information section ](http://localhost:8888/reference/classes/wp_query/#more-information). Could you try removing it?

Reply:
How does 
```
.wp-block-search div.awesomplete > ul {
	left: unset;
	inset-inline-start: 0;
}
```
affect the value here?

Reply:
Oh, wait. I guess you're referring to [this one](https://github.com/WordPress/wporg-developer/blob/trunk/source/wp-content/themes/wporg-developer-2023/src/style/style.scss#L192)
```
padding: 4px 6px;
```
Must be the code has been updated.

Reply:
I've just replaced the `padding: 4px 6px;` with ```padding-inline-start: 0.25em;
		padding-inline-end: 0.25em;```

You can look for `.wporg-has-embedded-code *:not(.wp-block-code):not(.wp-code-block-button-container) > code` elements on http://localhost:8895/reference/classes/wp_query/ to see if those affected are expected.

Reply:
I've removed it as well as other redundant styles in d04799c.